tests/Unit/Order/RowTest.php

```diff
@@ -10,14 +10,18 @@ class RowTest extends TestCase
     private function getJsonData(): array
     {
         return [
+            'orderRowSequence' => 10,
             'productId' => 123,
-            'name' => "test",
-            'quantity' => 1,
+            'productName' => "test",
```

Comment:
Can we get unique values in this test data? Just helps prevents highlighting errors such as return wrong parameter in a get method.